doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -548,19 +548,46 @@ Creating Hints
 
       .. warning::
 
-         The regexp matches the entire path. Most hints will start with a
-         leading `( _* )` to match the tail of the path. (Note that `(_*)`
-         misparses since `*)` would end a comment.)
+         These hints currently only apply to typeclass proof search and the
+         :tacn:`typeclasses eauto` tactic.
 
       .. warning::
 
-         There is no operator precedence during parsing, one can
-         check with :cmd:`Print HintDb` to verify the current cut expression.
+         The regexp matches the entire path. Most hints will start with a
+         leading `( _* )` to match the tail of the path.
 
-      .. warning::
+      .. knownissue:: Hint Cut regexp precedence
 
-         These hints currently only apply to typeclass proof search and the
-         :tacn:`typeclasses eauto` tactic.
+         :cmd:`Hint Cut` doesn't apply any operator precedence when
+         parsing regular expressions.  You can check how the cut
+         expression has been parsed with :cmd:`Print HintDb`.
```

Comment:
I think this is too much detail.  It's longer than the description of the command.  If this level of detail is given for every bug, it will make the document bloated.

Reply:
In addition, many of the issues are quite technical, making them hard to understand quickly.

You could say something like "Hint Cut doesn't apply any operator precedence when parsing regular expressions even though `Print HintDb` prints the expression as if `*` has higher precedence.  For example `[_* bar _* bar]` is interpreted and printed as `(_* bar _)* bar`, but it should be interpreted as `(_* bar (_*) bar)`, which is equivalent to `(_* bar _* bar)`.

Reply:
Are you talking about the inclusion of an example? My claim is that known issues should be documented (with workarounds) in the doc rather than in the bug tracker.

Reply:
I'm not sure that's realistic.  I stand by my comment:

> I think this is too much detail. It's longer than the description of the command. If this level of detail is given for every bug, it will make the document bloated.

Reply:
I guess we shouldn't (and we don't need) to get to that level of detail for every bug. But I don't see why I should backtrack on the addition of a very simple example of which you told me yourself you thought it was good. This is not going to make the documentation harder to read or harder to browse in any way.

Reply:
I thought it was a good writeup to include in the issue but too much for the doc.  Do what you think is best for now.  We'll figure out the best way to handle these as we get experience adding known issues to the doc.

Isn't it past your bedtime :-)?